apps/hiring-kanban/src/components/CardForms/CardHistory.tsx

```diff
@@ -24,7 +32,46 @@ function TitleComponent({ value }: CellProps<TCardHistoryCard>) {
   return <div className="text-sm truncate">{value || 'Untitled'}</div>;
 }
 
+function DoubleIconDisabled({
+  title,
+  updatePreview,
+  leftIcon,
+  rightIcon,
+}: {
+  title: string;
+  updatePreview: () => void;
+  leftIcon: boolean;
+  rightIcon: boolean;
+}) {
+  return (
+    <button type="button" title={title} disabled onClick={() => updatePreview}>
+      {leftIcon && (
+        <ChevronDoubleLeftIcon
+          aria-hidden="true"
+          className="absolute -mt-60 ml-10  h-8 w-8 text-stone-400 cursor-not-allowed"
+        />
+      )}
+      {rightIcon && (
+        <ChevronDoubleRightIcon
+          aria-hidden="true"
+          className="absolute -mt-60 h-8 w-8 text-stone-400 cursor-not-allowed"
+          style={{ marginLeft: '50rem' }}
```

Comment:
generally, if you find yourself reaching for margin of more than 2 or 3 em you should look for a different way to do it. Also with tailwind you shouldn't use 'style' unless you really need to, because its less efficient